google-cloud-storage/lib/google/cloud/storage/bucket/lifecycle.rb

```diff
@@ -174,7 +174,7 @@ def add_set_storage_class_rule storage_class,
           end
 
           ##
-          # Adds a SetStorageClass lifecycle rule to the Object Lifecycle
+          # Adds a Delete lifecycle rule to the Object Lifecycle
```

Comment:
This mistake should've been corrected in a different PR, but it was tiny, so I changed it here.